TheDailyWtf/Views/Home/Index.cshtml

```diff
@@ -1,6 +1,7 @@
 ﻿@model HomeIndexViewModel
 @{
     Layout = "~/Views/Shared/_Layout.cshtml";
+    SidebarWtfs = Model.GetSideBarWtfs();

```

Comment:
Will this work? As in, does anyone know if this is valid ASP.NET/MVC? Or, does it have to be like, on the model or something?? 

I also don't have a local working copy, and I'm not keen on spending an afternoon figuring out how to rollback a pull request using git, otherwise i'd just merge and publish to the test site.